ci/config/settings.json

```diff
@@ -1,3 +1,8 @@
 {
```

Comment:
Why would every test need his own image path? This should be by default on `/root/test_image.img`